pkgs/tools/security/sbomnix/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, coreutils
+, curl
+, gnugrep
+, gnused
+, gzip
+, graphviz
+, nix
+, reuse
+}:
```

```suggestion
{ lib
, fetchFromGitHub
, coreutils
, curl
, gnugrep
, gnused
, gzip
, graphviz
, nix
  # python libs
, reuse
, colorlog
, graphviz
, numpy
, packageurl-python
, pandas
, requests
, tabulate
}:
```

After the changes suggested in all-packages & the wrapper

---

Fixed in 0f1b2e7747bb9dea653ca76d107028bc25450b22, I hope I got your suggestion right.